tests/nlu/data_augmentation/test_augmenter.py

```diff
@@ -37,167 +35,202 @@ def test_augmenter_create_tokenizer(run: Callable[..., RunResult],):
     assert tokens == expected_tokens
 
 
-def test_augmenter_create_tokenizer_empty_config(run: Callable[..., RunResult],):
+@pytest.mark.xfail(raises=InvalidConfigException)
```

Comment:
I think we don't use this decorator anywhere else to test exceptions. Not sure how it would affect the test suite setup. I'd suggest taking the same approach for catching exceptions as the rest of the codebase does, for e.g. - https://github.com/RasaHQ/rasa/blob/793d13c6bcf81e2a70072999fcfffb0559059499/tests/nlu/test_train.py#L214

Reply:
Ah right, can do that. I've seen the decorator option on the `pytest` docs and thought it looked neater, though I can certainly change it.

Reply:
With the refactoring around re-using `get_tokenizer_from_nlu_config(...)`, I have removed that testcase as the re-used function does not raise any exceptions when no tokenizer can be found, but simply returns `None` (though I do raise an exception in the augmentation code if `get_tokenizer_from_nlu_config(...)` returns `None`).